grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy

```diff
@@ -161,6 +167,108 @@ class StockMovementApiController {
         forward(action: "read")
     }
 
+    def exportPickListItems = {
+        StockMovement stockMovement = stockMovementService.getStockMovement(params.id, "4")
+
+        List<PicklistItem> picklistItems = stockMovement?.pickPage?.pickPageItems?.inject([]) { result, pickPageItem ->
+            result.addAll(pickPageItem.picklistItems)
+            result
```

Comment:
Is this line required?

Reply:
it's to return the result

Reply:
when I was looking on some example it was not necessary, but i guess this is not a bad approach